packages/itwinui-react/src/core/Tile/TileNew.tsx

```diff
@@ -0,0 +1,646 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import React from 'react';
+import cx from 'classnames';
+import {
+  useSafeContext,
+  // useTheme,
+  SvgMore,
+  type PolymorphicComponentProps,
+  type PolymorphicForwardRefComponent,
+  StatusIconMap,
+  SvgCheckmark,
+  SvgNew,
+  LinkAction,
+  supportsHas,
+  polymorphic,
+} from '../utils/index.js';
+import '@itwin/itwinui-css/css/tile.css';
+import { ProgressRadial } from '../ProgressIndicators/index.js';
+import { DropdownMenu } from '../DropdownMenu/index.js';
+import { IconButton } from '../Buttons/index.js';
+// import { ProgressRadial } from '../ProgressIndicators';
+// import Tile from '.';
+
+const TileContext = React.createContext<
+  | {
+      status: 'positive' | 'warning' | 'negative' | undefined;
+      /**
+       * Whether the tile is selected or in "active" state.
+       * Gets highlighted and shows a checkmark icon near tile name.
+       */
+      isSelected?: boolean;
+      /**
+       * Whether the tile is "new". Tile name becomes bold and gets a new status icon.
+       */
+      isNew?: boolean;
+      /**
+       * Default tile variant or the folder layout.
+       * @default 'default'
+       */
+      variant?: 'default' | 'folder';
+      /**
+       * Whether the tile is expected to be interactable (i.e. `onClick`).
+       * It becomes focusable and gets on hover styling.
+       */
+      isActionable?: boolean;
+      /**
+       * Display a loading state.
+       * @default false
+       */
+      isLoading?: boolean;
+      /**
+       * Flag whether the tile is disabled.
+       *
+       * Note: This only affects the tile. You need to manually disable
+       * the buttons and other interactive elements inside the tile.
+       *
+       * @default false
+       */
+      isDisabled?: boolean;
+      setActionable: React.Dispatch<React.SetStateAction<boolean>>;
+      onClick?: React.MouseEventHandler<HTMLElement>;
+    }
+  | undefined
+>(undefined);
+TileContext.displayName = 'TileContext';
+
+type TileActionOwnProps = {}; // eslint-disable-line
```

Comment:
The comment on this line should be changed to the following 

`// eslint-disable-line @typescript-eslint/ban-types`

Also, I think it makes sense for the component props to come before the composition like it does here. Could you change this for the other components which list the props after the component?

Reply:
Yea this makes sense. I moved props to above components. Also updated comment.